### PR TITLE
Change git config files result to relative path

### PR DESCRIPTION
**Pull request by achiar99**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

In this pr we set the git configuration resource (resources that coming from github api) to be relative

### CODE REVIEW

File: checkov/json_doc/runner.py [hunk 1]
```diff
@@ -85,3 +85,9 @@ def import_registry(self):
 
     def require_external_checks(self):
         return True
+
+    @staticmethod
+    def _change_files_path_to_relative(report: Report):
+        for record in report.get_all_records():
+            record.file_path = record.file_path.replace(os.getcwd(), '')
+            record.resource = record.resource.replace(os.getcwd(), '')
```
Comment (Saarett) [inline checkov/json_doc/runner.py:93]: Is it OS agnostic?

**Review by Saarett — COMMENTED** [review]
Good job! A question in the code :)

File: checkov/json_doc/runner.py (hunk 1 above)
Comment (achiar99) [reply checkov/json_doc/runner.py:93]: @Saarett  what do u mean?

Comment (Saarett) [reply checkov/json_doc/runner.py:93]: Would it yield the same result for different operating systems? Windows, Linux....

Comment (Saarett) [reply checkov/json_doc/runner.py:93]: And what if the user would run it locally? Or is it not possible?

Comment (achiar99) [reply checkov/json_doc/runner.py:93]: it is possible. when the user run checkov cli. but in both cases `os.getcwd()` will be the same

**Review by Saarett — APPROVED** [review]

**Pull request merged**